metagpt/document.py

```diff
@@ -0,0 +1,255 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+@Time    : 2023/6/8 14:03
+@Author  : alexanderwu
+@File    : document.py
+"""
+from enum import Enum
+from pathlib import Path
+from typing import Optional, Union
+
+import pandas as pd
+from langchain.document_loaders import (
+    TextLoader,
+    UnstructuredPDFLoader,
+    UnstructuredWordDocumentLoader,
+)
+from langchain.text_splitter import CharacterTextSplitter
+from pydantic import BaseModel, Field
+from tqdm import tqdm
+
+from metagpt.config import CONFIG
+from metagpt.logs import logger
+from metagpt.repo_parser import RepoParser
+
+
+def validate_cols(content_col: str, df: pd.DataFrame):
```

Comment:
most content repeated with `metagpt/document_store/document.py`

Reply:
The two code files have different functionalities, and this has been annotated for clarification. metagpt/document_store/document.py is still under design; please ignore it for now.